cli/jira_config_gen/jira_config_gen.py

```diff
@@ -26,7 +27,7 @@ def __init__(
         server_url: str,
         token_path: str,
         output_file: str,
-        template_dir: str = "/templates",
+        template_dir: str = os.path.join("firewatch", "cli", "templates"),
```

Comment:
@myakove have you tested this? I am getting an error trying to render a template when using the `firewatch jira_config_gen` command.

`jinja2.exceptions.TemplateNotFound: jira.config.j2`

Reply:
Fixed

Reply:
@myakove did you forget to push your changes? I don't see any change within the last couple hours.

Reply:
:), pushed